cli/compose/loader/loader_multiple_test.go

```diff
@@ -0,0 +1,114 @@
+package loader
+
+import (
+	"testing"
+
+	"github.com/docker/cli/cli/compose/types"
+	"github.com/stretchr/testify/require"
+)
+
+/*
+func TestLoadTwoDifferentVersion(t *testing.T) {
+	configDetails := types.ConfigDetails{
+		ConfigFiles: []types.ConfigFile{
+			{Filename: "base.yml", Config: map[string]interface{}{
+				"version": "2.0",
+			}},
+			{Filename: "override.yml", Config: map[string]interface{}{
+				"version": "3.0",
+			}},
+		},
+	}
+	_, err := Load(configDetails)
+	require.Error(t, err)
+}
+*/
```

Comment:
i think we should remove commented code :)

Reply:
@ksouf oh right, I kept this one to discuss it with @dnephin and @shin- if we do support merging 2 different version on `docker-compose` or not.

Reply:
@vdemeester all right :)

Reply:
oh but it doesn't really make sense in the PR because it's not the one implementing parsing of v1/v2 compose files 😅

Reply:
It is not supported, it's an error to combine different versions.